Makefile

```diff
@@ -1,62 +1,14 @@
-SHELL:=/usr/bin/env bash -O nullglob
-
-XDG_CONFIG_HOME ?= $$HOME/.config
-FISH_CONFIG := $(XDG_CONFIG_HOME)/fish/config.fish
-
-FISHER_HOME := $(dir $(abspath $(lastword $(MAKEFILE_LIST))))
-FISHER_CONFIG := $(XDG_CONFIG_HOME)/fisherman
-FISHER_CACHE := $(FISHER_CONFIG)/cache
-FISHER_FILE := $(FISHER_CONFIG)/fishfile
-
-MAN := $(FISHER_HOME)/man
-MAN1 := $(wildcard $(MAN)/man1/*.md)
-DOCS := $(MAN1:%.md=%.1)
-
-INDEX := $(FISHER_CACHE)/.index
-VERSION = `cat $(FISHER_HOME)/VERSION`
-
-.PHONY: all test flush uninstall release
-
-all: $(FISH_CONFIG) $(FISHER_CACHE) $(FISHER_FILE) $(DOCS)
```

Comment:
Where did `make all` went? If it isn't here, isn't `make install` just the same as `make`?